src/transformations/correction/closed_ring.jl

```diff
@@ -0,0 +1,66 @@
+# # Closed Rings
+
+export ClosedRing
+
+# A closed ring is a ring that has the same start and end point. This is a requirement for a valid polygon (technically, for a valid LinearRing). 
+# This correction is used to ensure that the polygon is valid.
+
+# The reason this operates on the polygon level is that several packages are loose about whether they return LinearRings (which is correct) or LineStrings (which is incorrect) for the contents of a polygon.
+# Therefore, we decompose manually to ensure correctness.
+
+# ## Example
+#=
+
+Many polygon packages do not close their polygons, which makes them invalid according to strict specification.  
```

Comment:
I would add that this is also depended upon for many geometry algorithms, which is why this is usually the specification.